src/AuthenticationApp/firebase.js

```diff
@@ -0,0 +1,41 @@
+import firebase from "firebase/app";
```

Comment:
Will having 2 firebase setup files (Shaina's and yours) interfere with eachother? If not its fine, but I just want to make sure.

Reply:
From what I've seen they won't, but If you find something that says otherwise I'll merge the stuff I need into her file.